web/src/components/Button.tsx

```diff
@@ -3,10 +3,10 @@ import { twMerge } from 'tailwind-merge';
 
 type SizeOptions = 'sm' | 'md' | 'lg' | 'xl';
 
-interface ButtonProps {
+export interface ButtonProps {
   icon?: React.ReactNode;
   children?: React.ReactNode;
-  disabled?: boolean;
+  isDisabled?: boolean;
```

Comment:
There was a ESLint rule that complained about this when I tried to upgrade it, figured I might as well change it now rather than later but I'd be happy to change it back if you think it should be a separate PR.